### PR TITLE
secound section

### PR DESCRIPTION
**Pull request by lina-jamal**

### CODE REVIEW

**Review by mFlaifel — APPROVED** [review]

File: CSS/style.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+body { 
+
+}
```
Comment (Mu7ammadAbed) [inline CSS/style.css]: Empty styling

File: CSS/style.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+.our_services .container{
+    display: flex;
+
+}
+.our_services .container img{
+    width: 25%;
+    height: 25%;
+  
+
+}
```
Comment (Mu7ammadAbed) [inline CSS/style.css]: need a new line

File: CSS/style.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+.our_services{
+   
+    width: 75%;
+ 
+}
+.our_services .container{
+    display: flex;
+
+}
+.our_services .container img{
```
Comment (Mu7ammadAbed) [inline CSS/style.css]: Please convert the dot "." into hash "#"

**Review by Mu7ammadAbed — CHANGES_REQUESTED** [review]

**Review by Mu7ammadAbed — CHANGES_REQUESTED** [review]
Please resolve the conflicts.

**Review by Mu7ammadAbed — APPROVED** [review]

**Review by rehabas — APPROVED** [review]

**Pull request merged**